.github/workflows/github-ci.yml

```diff
@@ -20,22 +20,34 @@ jobs:
                   java-version: '11.x'
             - name: Install node.js packages
               run: npm install
-            - name: Run backend test
-              run: |
-                  chmod +x mvnw
-                  ./mvnw -ntp clean verify -P-webpack
-            - name: Run frontend test
-              run: npm run test
-            - name: Sonar analysis
-              if: github.repository == 'jhipster/jhipster-control-center' && github.ref == 'refs/heads/master'
-              run: |
-                  ./mvnw -ntp --batch-mode initialize org.jacoco:jacoco-maven-plugin:prepare-agent sonar:sonar \
-                  -Dsonar.host.url=https://sonarcloud.io \
-                  -Dsonar.projectKey=jhipster_jhipster-control-center \
-                  -Dsonar.organization=jhipster \
-                  -Dsonar.login=$SONAR_TOKEN
-              env:
-                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+            #- name: Run backend test
```

Comment:
just to know why did you comment this part @avdev4j ?

Reply:
It's a mistake. I did it to do quick tests. I'll fix it tomorrow.